src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8724,6 +8724,41 @@ paths:
           schema:
             $ref: '#/definitions/Error'
   '/nodes/{nodeId}/rule-sets/{ruleSetId}':
+    get:
+      x-alfresco-since: "7.3.0"
+      tags:
+        - rules
+      summary: Retrieve a rule set from a folder
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.3 and newer version.
+
+        Gets information about a rule set **ruleSetId** that is used (either owned or linked) by the folder **nodeId**.
```

Comment:
Could be wrong but shouldn't this be "folder**'s** **nodeId**". Other than that, LGTM

Reply:
The API Explorer tends to treat ids as names - e.g.

> /nodes/{nodeId}
> Get information for node nodeId.

> /nodes/{nodeId}/children
> Gets a list of children of the parent node nodeId.

I think this is consistent with the general idea.